networker/networker.go

```diff
@@ -997,11 +1047,12 @@ func (netw *Combined) unsetAllowlist() error {
 	}
 
 	for _, rule := range []string{
-		"allowlist_subnets",
-		"allowlist_subnets_forward",
-		"allowlist_forward_related",
-		"allowlist_ports_tcp",
-		"allowlist_ports_udp",
+		"allowlist_subnets_in",
+		"allowlist_subnets_out",
```

Comment:
missing to remove `allowlist_forward_related` and `allowlist_subnets_forward` from FW

Reply:
Fixed.